src/site/twitch.tv/modules/chat/ChatModule.vue

```diff
@@ -142,6 +142,12 @@ export const config = [
 		},
 		defaultValue: 1,
 	}),
+	declareConfig("chat.show_emote_modifiers", "TOGGLE", {
+		path: ["Chat", "Style"],
+		label: "Show Emote Modifiers",
+		hint: "Show BTTV and FFZ emote modifiers (!w, ffzHyper, etc.)",
```

Comment:
Since we don't actually support these yet, we should specify that this only governs whether or not to show text pollution.

```suggestion
		hint: "Show text pollution from BTTV and FFZ emote modifiers (!w, ffzHyper, etc.). Modifiers are not supported yet, this setting only affects the display of their text form",
```